### PR TITLE
Added nginx PHP-FPM configuration

### PR DESCRIPTION
**Pull request by sammcj**
Updated nginx config to correctly include upstream configuration for php-fpm.

I can't see any reason to give instructions for both with and without SSL, it should be easy enough to remove it if someone *really* wanted to for testing reasons.

Apologies for missing this.

### CODE REVIEW

**Comment by iMstr** [comment]
I think there is one location rule to be added. eg. /setup returns a 404.

File: INSTALL.md [hunk 1]
```diff
@@ -133,51 +133,58 @@ Restart Apache and you're done!
...
+    ssl_ciphers "ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:kEDH+AESGCM:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA256:DHE-RSA-AES256-SHA256:DHE-DSS-AES256-SHA:DHE-RSA-AES256-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:AES:CAMELLIA:DES-CBC3-SHA:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!MD5:!PSK:!aECDH:!EDH-DSS-DES-CBC3-SHA:!EDH-RSA-DES-CBC3-SHA:!KRB5-DES-CBC3-SHA";
+    ssl_buffer_size 1400; # 1400 bytes, within MTU - because we generally have small responses. Could increase to 4k, but default 16k is too big
+
+    location / {
+        add_header Strict-Transport-Security max-age=15768000;
+    }
+
+    location ~ \.php$ {
+                include fastcgi_params;
+    fastcgi_pass unix:/var/run/php5-fpm.sock;
```
Comment (joecohens) [inline INSTALL.md]: Can you fix the indentation here? Thanks!

Comment (joecohens) [inline INSTALL.md]: Well I guess indentation in general all 4 spaces.

**Comment by sammcj** [comment]
@iMstr is there documentation on what locations the app needs? I didn't see anything specified for apache?

File: INSTALL.md (hunk 1 above)
Comment (sammcj) [inline INSTALL.md]: I used 4 spaces as that's what was used in the apache example, would you like me to change it to 2 spaces?

**Comment by iMstr** [comment]
@sammcj there is a .htaccess in Public/ but I found out what is needed. You need to ass `try_files $uri /index.php$is_args$args;` to Location /

**Comment by sammcj** [comment]
@iMstr ah good old .htaccess eh, I haven't used Apache in quite some time, added.

**Comment by iMstr** [comment]
+1

**Pull request merged**